examples/sample-data/put_customer_order_items.py

```diff
@@ -0,0 +1,186 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from decimal import Decimal
+import boto3
+import csv
+
+dynamo = boto3.resource('dynamodb')
+
+def put_customer_item(pk, sk, customer_id, customer_fname, customer_lname, customer_email, customer_addresses):
+    
```

Comment:
```suggestion
```

Minor nits in this and suggestions below to remove extra newlines to be consistent with the Google Python style guide.